### ISSUE TITLE
Open new in new tab and get canvas link should not exist on new "sections" & "headings"

### ISSUE THREAD

**Issue by a20vicad** [HGustavs/LenaSYS#12364]
Title says it all
Related to #12348
![image](https://user-images.githubusercontent.com/102599327/167629035-29b51fb2-d547-45d5-a73b-da05d13ba1bd.png)

**Comment by a17nikjo** [comment]
Will do futher research for now and start working on a solution once the weekly merge is done.

**Comment by a17nikjo** [comment]
@a20vicad i can not find the issue you are talking about, was this tested before #12348 was merged? Since it seems that issue resolved this problem once it was merged into the new week.
